### PR TITLE
feature: icky 765

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/layout/Login/LoginContent/LoginContent.spec.tsx [hunk 1]
```diff
@@ -7,8 +7,6 @@ import userEvent from '@testing-library/user-event'
 
 import * as stories from './LoginContent.stories' // import all stories from the stories file
 
-import type { UserEvent } from '@testing-library/user-event/dist/types/setup/'
```
Comment (kibo-kevinwatts) [inline components/layout/Login/LoginContent/LoginContent.spec.tsx]: don't modify these core components test files inside the fork, add in the core repo

File: hooks/queries/useProductsQueries/useProductsQueries.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ const fetchProductSearch = async (searchParams: CategorySearchParams) => {
 export const useProductsQueries = (productCodes: ProductCodes[]): UseProductsResponse => {
   const productCodeFilter: Array<string> = []
   productCodes?.forEach((code) => {
-    productCodeFilter.push(`productCode eq ${code?.productCode}`)
+    productCodeFilter.push(`productCode eq ${code}`)
```
Comment (kibo-kevinwatts) [inline hooks/queries/useProductsQueries/useProductsQueries.ts]: try to avoid modifying the core hooks

File: lib/getters/amplienceGetters.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
+import getConfig from 'next/config'
+
+const { publicRuntimeConfig } = getConfig()
+
+const formatLinkData = (links: any) => {
```
Comment (kibo-kevinwatts) [inline lib/getters/amplienceGetters.ts]: are these getters still needed? if so we should move under the `/cms` folder to keep everything togeter

File: tsconfig.json [hunk 4]
```diff
@@ -23,6 +23,6 @@
       "@/hooks/*": ["hooks/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "cms/index.js"],
```
Comment (kibo-kevinwatts) [inline tsconfig.json]: can we just change cms/index to typescript and avoid changing the ts config?

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 4 above)
Comment (kibo-chandradeeptalaha) [reply tsconfig.json]: Done

File: lib/getters/amplienceGetters.ts (hunk 3 above)
Comment (kibo-chandradeeptalaha) [reply lib/getters/amplienceGetters.ts]: Done

**Review by kibo-geetanshuchhabra — APPROVED** [review]

File: components/layout/Login/LoginContent/LoginContent.spec.tsx (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply components/layout/Login/LoginContent/LoginContent.spec.tsx]: Resolved

File: hooks/queries/useProductsQueries/useProductsQueries.ts (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply hooks/queries/useProductsQueries/useProductsQueries.ts]: Resolved

**Review by TriptiVermaXP — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**